flask_resources/views/collections.py

```diff
@@ -7,10 +7,10 @@
 
 """Flask Resources module to create REST APIs."""
 
+from flask import request
```

Comment:
Moderate: I am not a fan of this. Sort of why the **resource**_requestctx exists... and now the resource is accessing `request`... I think it should be passed in `resource_requestctx`. Otherwise, there is no point on having it.

If blocking, just open an issue about it, so it is tackled.